examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
I am not sure how to fix this. From what I can tell MySQL is going to compile in /MT unless it is overridden by the end user with WINDOWS_RUNTIME_MD, which does not seem to be an accepted practice from the MySQL team. Therefore anything compiled to use MySQL will have to be /MT as well. Unless there is a way to force that from the library then the end user will have to know that they need todo this.

The only other thing I can think todo is modify MySQL to somehow detect the host applications run-time library setup and set WINDOWS_RUNTIME_MD accordingly.